package.json

```diff
@@ -31,7 +31,7 @@
     "reinstall": "rm -rf node_modules && yarn cache clean && yarn install"
   },
   "dependencies": {
-    "@expo/react-native-action-sheet": "^1.1.0",
+    "@expo/react-native-action-sheet": "^3.4.0",
```

Comment:
I updated `@expo/react-native-action-sheet` dependency to the [latest version 3.4.0](https://www.npmjs.com/package/@expo/react-native-action-sheet/v/3.4.0) because `title` option for the method `showActionSheetWithOptions` is not availabe for the outdated version [`1.1.0`](https://www.npmjs.com/package/@expo/react-native-action-sheet/v/1.1.0).

Reply:
According to the documentation, there is a difference in the dependencies of V2 and V3. V2 has same dependencies as V1 but V3 has different dependencies. We can also use V2 because it looks more close to 1.1.0 version and doesn't add any new dependency to the project.

Here is the documentation of all three versions:
https://www.npmjs.com/package/@expo/react-native-action-sheet/v/1.1.0
https://www.npmjs.com/package/@expo/react-native-action-sheet/v/2.0.1
https://www.npmjs.com/package/@expo/react-native-action-sheet/v/3.4.0

Reply:
The dependencies are fine. Let's use the latest version.

I'm still curious, though: why did they increase the major version number? That generally means there's some change that they expected to break some existing code that uses the library. It'd be good to know what that change was, so we can check if it affects us and there's something we need to do to adjust for it.